src/mediapipe_internal/mediapipegraphexecutor.hpp

```diff
@@ -296,7 +296,7 @@ class MediapipeGraphExecutor {
                             numberOfPacketsCreated),
                         "partial deserialization of subsequent requests", isSuccess);
                 } else {
-                    OVMS_WRITE_ERROR_ON_FAIL_AND_CONTINUE(pstatus, "validate subsequent requests", isSuccess);
+                    OVMS_WRITE_ERROR_ON_FAIL_AND_CONTINUE(std::move(pstatus), "validate subsequent requests", isSuccess);
```

Comment:
Is it safe to use std move here? In case this macro will ever use first parameter twice, the pstatus will get moved twice, which is undefined behavior @atobiszei